src/App/Fossa/Analyze.hs

```diff
@@ -108,7 +110,25 @@ analyze basedir destination override unpackArchives = runFinally $ do
 
   let projects = mkProjects closures
       result = buildResult projects failures
- 
+
+  for_ projects $ \project -> do
+    let bestStrategy :: ProjectStrategy
+        bestStrategy = NE.head . projectStrategies $ project
+
+        bestStrategyName :: Text
+        bestStrategyName = projStrategyName $ bestStrategy
+
+        bestStrategyDepCount :: Int
+        bestStrategyDepCount = Graphing.size . projStrategyGraph $ bestStrategy
+
+    logInfo $
+      "Found " <> pretty bestStrategyName
+        <> " project at "
+        <> pretty (fromAbsDir (projectPath project))
+        <> " with "
+        <> pretty bestStrategyDepCount
+        <> " dependencies"
```

Comment:
I'd prefer to see this functionality a little more restructured so we can improve the color in the output later.  Some things I think would make this better long term are:

* `map` the projects into some data structure that represents the data we want to output.
* have some `render` function that turns that data structure into the properly-typed argument for `logInfo` (if there's already a typeclass, let's go with that)
* moving all of that into a separate function for readability.  Specifically, I think that `analyze` should only have 1 new line of code.